dev/tests/integration/testsuite/Magento/GiftMessage/_files/guest/quote_with_item_message.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
Please, do not use deprecated classes in the fixtures. Try to build fixtures according to the best practices. In the future, we will need less effort for the refactoring once the deprecated methods will be removed

Reply:
Hi @rogyar .
I have refactored fixture. Could you check please and parhaps advice me something ?
Thanks